draft-zsf.md

```diff
@@ -0,0 +1,154 @@
+
+The `ZSF_BALANCE[H]` for a block at height `H` can be calculated given a value of `ZSF_BALANCE[H-1]` and the set of transactions contained in that block. First, the `ZSF_DEPOSIT[H]` is calculated based solely on `ZSF_BALANCE[H-1]`. This is subtracted from the previous block's balance to be distributed as part of the block reward. Second, the sum of all the `ZSF_DEPOSIT` fields of all transactions in the block is added to the balance.
+
+It is safe and consistent to treat older transactions using pre-Sustainability Fund formats as if they have this field implicitly present with a value of 0 where that simplifies designs or implementations.
+
+Note: this field is not generally considered an "output" because it differs from other outputs in several significant ways:
+
+- All `ZSF_DEPOSIT` fields contribute to a single global balance rather than a user-specific output state,
+- Consensus validation can account for this field by updating `ZSF_BALANCE[H]` and do not need to track any transaction field specific state after this accounting in contrast to e.g. UTXOs which must be tracked until spent.
+- This field does not contribute to the transaction graph of senders or recipients whether transparent or protected by cryptography.
+
+### `ZSF_DEPOSIT` Requirements
+
+- There MUST be only one `ZSF_DEPOSIT` field per transaction
+- ZIP 225 [3] MUST be updated to include `ZSF_DEPOSIT`. ZIP 244 MAY be updated as well.
+- Separate programming language implementations (C++, Rust, etc) MUST guarantee that the calculations described above are consistent
+
+# Rationale
+
+All technical decisions in this ZIP are balanced between the necessary robustness of the ZSF mechanics, and simplicity of implementation. 
+
+## `ZSF_BALANCE` as node state
+
+Tracking the `ZSF_BALANCE` value as a node state using the above formula is very simple in terms of implementation, and should work correctly given that the node implementations calculate the value according to the specifications.
+
+### Alternative: `ZSF_BALANCE` as block header commitment
+
+An alternative to node state could be to include the `ZSF_BALANCE` field as a block header and require a block header commitment.
+
+Requiring block-header-rooted commitments of global fund balances such as the Sustainability Fund ensures that any consensus deviating bugs in accounting of this balance are immediately detected in the earliest impacted block. It also removes some of the need for explorer sites and other analytics services from tracking this value independently, assuming the committed value is made available by common APIs. This helps ensure that all explorers track and report the correct value.
+
+## `ZSF_DEPOSIT` as explicit transaction field
+
+An explicit value distinguishes the ZEC destined to Sustainability Fund deposits from the implicit transaction fee. Explicitness also ensures any arithmetic flaws in any implementations are more likely to be observed and caught immediately.
+
```

Comment:
Some commitments start in the activation block, and some in the block after, so we need to be precise here.

```suggestion

# Deployment

This ZIP is proposed to activate with Network Upgrade (TODO ZIP editors).

The `ZsfBalanceAfter` node state commitment will be included in the `hashBlockCommitments` chain starting with the network upgrade activation block.
```